app/utils/computed-helpers.js

```diff
@@ -46,9 +46,9 @@ export const computedSegmentedLink = function(property) {
 export const computedDateTimeSplit = function(property, segmentFormat) {
   return computed(property, {
     get() {
-      if (property === 'endsAt' && segmentFormat === 'date') {
-        return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
-      }
+      // if (property === 'endsAt' && segmentFormat === 'date') {
+      //   return moment(this.get(property)).add(1, 'days').format(getFormat(segmentFormat));
+      // } The following line was adding one extra day to the endsAt Attribute . Commenting it to fix the issue .
```

Comment:
@CosmicCoder96 This fixes the issue but I want to know why was this placed in the first place? Does it has any other utility which maybe undermined by removing this?

Reply:
Not sure. Though storing the wrong date is definitely wrong. Let's see in the test run, this week after merge.